### ISSUE TITLE
System.DllNotFoundException: /system/lib/libsqlite.so

### ISSUE THREAD

**Issue by aysecnblt** [oysteinkrog/SQLite.Net-PCL#380]
My app fine for Android7 lower versions.But it doesn't work on Android7+.Please Help me

They didn't work:

SQLiteNetExtensions
SQLite.Net.Platform.XamarinAndroidN  (I can't install)(Package SQLite.Net.Platform.XamarinAndroidN 3.1.1 is not compatible with monoandroid81 (MonoAndroid,Version=v8.1). Package SQLite.Net.Platform.XamarinAndroidN 3.1.1 supports: net (.NETFramework,Version=v0.0))

**Comment by jdcalderin** [comment]
Hi @aysebursa   i have the same problem, you could fix?

**Comment by aysecnblt** [comment]
yes.i fixed today.Firstly, Uninstall all sqlite packages. install 'sqlite-net-pcl' (by Frank A. Krueger) latest version for all projects. Easy to use.  @jdcalderin 

https://www.nuget.org/packages/sqlite-net-pcl/

**Comment by mfranc28** [comment]
Works like a charm.

**Comment by heltonspr** [comment]
Hi, i have the same problem, you could fix?
